anko/library/generated/sdk25/src/Properties.kt

```diff
@@ -122,3 +122,12 @@ var android.widget.Toolbar.titleResource: Int
     @Deprecated(AnkoInternals.NO_GETTER, level = DeprecationLevel.ERROR) get() = AnkoInternals.noGetter()
     set(v) = setTitle(v)
 
+var android.widget.TextView.textAppearance: Int
+	@Deprecated(AnkoInternals.NO_GETTER, level = DeprecationLevel.ERROR) get() = AnkoInternals.noGetter()
```

Comment:
Could you explain why did you add this in `sdk25` folder? It already exists in `sdk23` version of `Properties` file.